crates/compute/src/cpu/alloc.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2025 Irreducible Inc.
+
+use std::fmt::Debug;
+
+use bytemuck::zeroed_vec;
+
+use crate::alloc::HostBumpAllocator;
+
+pub struct CpuComputeAllocator<F> {
+	data: Vec<F>,
+}
+
+impl<F> CpuComputeAllocator<F>
+where
+	F: Sync + Debug + Send + 'static,
+{
+	pub fn into_inner(&mut self) -> HostBumpAllocator<'_, F> {
```

Comment:
The naming seems a bit weird to me, don't quite understand why "inner", usually "inner" means that some object is stored inside. Maybe `to_bump_allocator`?